app/src/main/res/values/strings.xml

```diff
@@ -160,8 +160,8 @@
     <string name="tx_list_creation">Safe created</string>
     <string name="tx_list_receive">Receive</string>
     <string name="tx_list_send">Send</string>
-    <string name="tx_list_label_type_next">Next</string>
-    <string name="tx_list_label_type_queued">Queued</string>
+    <string name="tx_list_label_type_next">NEXT TRANSACTION</string>
```

Comment:
There could be a conflict section under `NEXT` I think 🤔 , if there are many transactions proposed with the current safe nonce. I think that is beyond the scope of this PR so product can look at the app string values with real life data and decide.

My point being, is that there may be more than 1 transaction here, so it would be `Next transactions` maybe? Anyhow, not something we should look into right now.

Reply:
I think the reasoning for singular is that only one of these will be executed?

I will clarify in slack with Tobias

Reply:
Just saw that, good call 👍